src/main/java/com/pretz/dyntreeserver/service/exceptions/IncorrectPasswordException.java

```diff
@@ -0,0 +1,10 @@
+package com.pretz.dyntreeserver.service.exceptions;
+
+import com.pretz.dyntreeserver.domain.User;
+
+public class IncorrectPasswordException extends RuntimeException {
+
+    public IncorrectPasswordException(User user) {
```

Comment:
I think i'd be better to just pass here a `String userName`. IMO `IncorrectPasswordException` doesn't have to know about a `User` class, the less dependencies, the better :-)

Reply:
Ok